client/src/templates/Challenges/classic/lower-jaw.tsx

```diff
@@ -167,6 +167,13 @@ const LowerJaw = ({
     if (onAttempt) onAttempt();
   };
 
+  const onSubmitButtonClick = () => {
+    if (submitButtonRef.current) {
+      submitButtonRef.current.disabled = true;
+    }
+    debounce(submitChallenge, 2000);
```

Comment:
```suggestion
    debounce(submitChallenge, 2000)();
```

The above change will make the debounce to work.

Debounce returns a function but here the returned function isn't executed so the challenge doesn't get submitted.